aldjemy/core.py

```diff
@@ -89,7 +110,24 @@ def __init__(self, pool, alias):
 
         self.alias = alias
         self.wrap = False
-        # pool.dispatch.first_connect.exec_once(self.connection, self)
+        # Replace sqlalchemy's handler with ours
+        if not pool._aldjemy_handlers_init:
+            # we assume it's the last one
+            previous_handler_wrapper = pool.dispatch.connect.listeners.pop()
+            assert (
+                previous_handler_wrapper.__name__ == "go"
+            )  # wraped by _once_unless_exception=True
+            handler = previous_handler_wrapper.__closure__[0].cell_contents
+            assert handler.__name__ == "first_connect"
```

Comment:
```suggestion
            assert handler.__name__ == "first_connect", (handler, handler.__name__)
```
Just in case this gets hit.  Also above maybe.
In general `repr(…)` could be used maybe.

Reply:
That's the repr output
```
<function create_engine.<locals>.first_connect at 0x7efd939d3940>
```

That was a good idea, but the object id kind of makes it difficult to use, or maybe we should use ?

```python
assert repr(handler).startswith('<function create_engine.<locals>.first_connect at'), repr(handler)
```